samples/boards/nordic/coresight_stm/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&cpuppr_vpr {
+	status = "okay";
+};
+
+&cpuppr_ram3x_region {
+	status = "okay";
+};
+
+&cpuflpr_vpr {
+	status = "okay";
+};
```

Comment:
why aren't nordic-flpr+ppr snippets used?

Reply:
First version was using snippets. We encountered side effects resulting from snippet enabling uart nodes.

For example, nordic-ppr "reserves" uart135
https://github.com/zephyrproject-rtos/zephyr/blob/main/snippets/nordic-ppr/boards/nrf54h20dk_nrf54h20_cpuapp.overlay#L10
While uart135 is a default uart instance for cpurad. As a result we had conflict in UICRs and Secure Domain FW wasn't booting cores.

Reply:
should minimal ppr/flpr snippets be present for scenarios where UART is not needed? Do we envision other scenarios where this will be needed?

Reply:
Whoever tries to use both Radio core and PPR, will face this problem.
Radio core uses uart135 as a default console:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad.dts#L22
Snippet 'nordic-ppr' assigns uart135 to PPR:
https://github.com/zephyrproject-rtos/zephyr/blob/main/snippets/nordic-ppr/boards/nrf54h20dk_nrf54h20_cpuapp.overlay#L11
Also, @nordic-krch found that snippet is applied as a last so it's impossible to override uart135 state in a board overlay.

Reply:
nordic-ppr and nordic-flpr are simple snippets with 1 or 2 entries in the cpuapp overlay so i just moved them to here. When STM becomes a standard to be used for console output in nrf54h then maybe uart135 and uart120 nodes may be removed from those snippets. I'm affraid that more snippets will just cause trouble with too many options for the user.